finetuner-workflow/finetune-workflow.yaml

```diff
@@ -68,11 +68,11 @@ spec:
     # range is allowed to fill out the current `context`, and then the entire
     # range is used to start the next `context`.
     - name: boundary_token
-      value: '\n'
```

Comment:
Does the finetuner work with the double quotation marks?

Reply:
Yes, I will change it back